src/main/java/hotelreservation/domain/Users.java

```diff
@@ -0,0 +1,59 @@
+package hotelreservation.domain;
+
+import javax.persistence.*;
+
+@Entity
+@Table(name="users")
+public class Users {
+
+    @Id
+    @GeneratedValue( strategy=GenerationType.AUTO )
+    private int idusers;
```

Comment:
https://upload.wikimedia.org/wikipedia/commons/thumb/c/c8/CamelCase_new.svg/1200px-CamelCase_new.svg.png

Reply:
HAHA I did that because you named the tables that way big guy.  The program couldn't handle the upper lower case very well.